applications/ChimeraApplication/tests/flow_over_cross_fractional_step/flow_over_cross_fractional_step.json

```diff
@@ -49,13 +49,40 @@
         },
         "echo_level": 0,
         "compute_reactions": false,
+        "pressure_linear_solver_settings":  {
+            "solver_type"                    : "amgcl",
+            "max_iteration"                  : 200,
+            "tolerance"                      : 1e-6,
+            "provide_coordinates"            : false,
+            "smoother_type"                  : "damped_jacobi",
+            "krylov_type"                    : "gmres",
+            "gmres_krylov_space_dimension"   : 50,
+            "use_block_matrices_if_possible" : false,
+            "coarsening_type"                : "aggregation",
+            "scaling"                        : false,
+            "verbosity"                      : 0
+        },
+        "velocity_linear_solver_settings": {
+            "solver_type"                    : "amgcl",
+            "max_iteration"                  : 200,
+            "tolerance"                      : 1e-6,
+            "provide_coordinates"            : false,
+            "smoother_type"                  : "damped_jacobi",
+            "krylov_type"                    : "gmres",
+            "gmres_krylov_space_dimension"   : 50,
+            "use_block_matrices_if_possible" : false,
+            "coarsening_type"                : "aggregation",
+            "scaling"                        : false,
+            "verbosity"                      : 0
+        },
         "volume_model_part_name": "FluidModelPart",
         "skin_parts": ["AutomaticInlet2D_inlet","Outlet2D_outlet","NoSlip2D_top","NoSlip2D_bottom","NoSlip2D_cross"],
         "no_skin_parts": [],
         "time_stepping": {
             "automatic_time_step": false,
             "time_step": 0.05
-        }
+        },
+        "reform_dofs_at_each_step":true
```

Comment:
this is needed as soon as the patch moves right?

Reply:
exactly ! That was the problem. And I had to update the results because I changed the linear solver settings !